apps/src/p5lab/spritelab/libraries/PoemBotLibrary.js

```diff
@@ -23,7 +23,7 @@ export default class PoemBotLibrary extends CoreLibrary {
       lines: [],
       font: {
         fill: 'black',
-        stroke: 'black',
+        stroke: 'rgba(0,0,0,0)',
```

Comment:
if we remove the `stroke` key-value pair from this object, does it default to a transparent stroke, or something different?

Reply:
Good question. I wasn't confident enough that it definitely wasn't being set elsewhere or by default. I could see it be unnecessary or it being just an extra layer of certainty/readability.

Reply:
that's fair -- looking through this file, i think our code can handle an undefined stroke just fine, but i'm not sure what p5 does if we set the stroke to undefined:
https://github.com/code-dot-org/code-dot-org/blob/dbf250e8bec2860bda142ade0ec12c9ab61d555a/apps/src/p5lab/spritelab/libraries/PoemBotLibrary.js#L353

Reply:
@madelynkasula Could you help me troubleshoot this build fail?

Reply:
it's the same failure as your previous PR -- maybe this branch wasn't updated with the latest changes from staging? (since pulling latest staging, merging it into your feature branch, and pushing fixed the build on that pr.)

[cucumber test log](https://cucumber-logs.s3.amazonaws.com/circle/10132/Chrome_learning_platform_level_types_contained_levels_eyes_output.html?versionId=DL_LpsIrb3jBFC6q9YJrz0bJeloUaGGv) (only available for ~24 hours)
<img width="1415" alt="Screen Shot 2021-09-09 at 2 46 26 PM" src="https://user-images.githubusercontent.com/9812299/132766914-4ae495df-2c86-4c40-a267-d41e76750077.png">

Reply:
Okay, I tried merging the staging branch into my branch and pushing again.

Reply:
@madelynkasula Today I'm seeing that build is passing but DTS is not. On #developers I see "DTS: no". Is that related?

Reply:
@madelynkasula Ready to merge!

Reply:
>  On #developers I see "DTS: no". Is that related?

nope! this does mean you shouldn't merge, but is usually because we're closing the pipeline due to an issue or to make sure it's clear for a specific reason